charts/incubator/amd-automatic-music-downloader/questions.yaml

```diff
@@ -0,0 +1,567 @@
+        - variable: POSTPROCESSTHREADS
+          label: "Post Process Threads"
+          description: "Controls number of threads used for Format conversion and replay gain tagging"
+          schema:
+            type: int
+            default: 1
+  # Include{containerConfig}
+
+  - variable: service
+    group: "Networking and Services"
+    label: "Configure Service(s)"
+    schema:
+      additional_attrs: true
+      type: dict
+      attrs:
+        - variable: main
+          label: "Main Service"
+          description: "The Primary service on which the healthcheck runs, often the webUI"
+          schema:
+            additional_attrs: true
+            type: dict
+            attrs:
+              # Include{serviceSelector}
+              - variable: main
+                label: "Main Service Port Configuration"
+                schema:
+                  additional_attrs: true
+                  type: dict
+                  attrs:
+                    - variable: port
+                      label: "Port"
+                      description: "This port exposes the container port on the service"
+                      schema:
+                        type: int
+                        default: 8688
+                        required: true
+                    - variable: advanced
+                      label: "Show Advanced settings"
+                      schema:
+                        type: boolean
+                        default: false
+                        show_subquestions_if: true
+                        subquestions:
+                          - variable: protocol
+                            label: "Port Type"
+                            schema:
+                              type: string
+                              default: "HTTP"
+                              enum:
+                                - value: HTTP
+                                  description: "HTTP"
+                                - value: "HTTPS"
+                                  description: "HTTPS"
+                                - value: TCP
+                                  description: "TCP"
+                                - value: "UDP"
+                                  description: "UDP"
+                          - variable: nodePort
+                            label: "Node Port (Optional)"
+                            description: "This port gets exposed to the node. Only considered when service type is NodePort, Simple or LoadBalancer"
+                            schema:
+                              type: int
+                              min: 9000
+                              max: 65535
+                          - variable: targetPort
+                            label: "Target Port"
+                            description: "The internal(!) port on the container the Application runs on"
+                            schema:
+                              type: int
+                              default: 80
```

Comment:
If you disable the service (aka do not give it a GUI), then you need to completely remove it here.
```suggestion

```